tests/functional/modules/test_zos_mvs_raw_func.py

```diff
@@ -51,6 +51,7 @@ def test_failing_name_format(ansible_zos_module):
 def test_disposition_new(ansible_zos_module):
     try:
         hosts = ansible_zos_module
+        DEFAULT_DATA_SET = get_tmp_ds_name(hosts)
```

Comment:
Lower case DEFAULT_DATA_SET, use upper case only for global (top level) variables.

I won't repeat this , but do so for the entire class please, thank you.